packages/components/src/theme/search.scss

```diff
@@ -225,3 +225,7 @@
     font-weight: bold;
     padding-bottom: 5px;
 }
+
+.filter-results__message {
```

Comment:
is this still needed with the move of the "all generations" text to the page subtitle?

Reply:
Removed.